applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -140,6 +144,9 @@ User interface
 Button 1:
    .. include:: /includes/matter_sample_button.txt
 
+Button 2:
+   If we are using extend functionality, we can change the state of smart plug device by pressing it.
```

Comment:
Avoid the "we" form, also extend functionality does not say much.
```suggestion
   In case the Matter smart plug functionality is used, you can change the state of smart plug device by pressing it.
```